sdk/examples/intkey_rust_cli/src/submitbatch.rs

```diff
@@ -0,0 +1,73 @@
+use hyper;
+use hyper::Method;
+use hyper::client::{Client, Request};
+use hyper::header::{ContentLength, ContentType};
+use futures::{future, Future};
+use futures::Stream;
+use tokio_core;
+
+use sawtooth_sdk::messages::batch::BatchList;
+
+use errors::IntkeyError;
+use protobuf::Message;
+
+
+
+pub fn submit_batch_list(url: Option<&str>, batch_list: &BatchList) -> Result<(), IntkeyError> {
+    let url = match url {
+        Some(v) => String::from(v) + "/batches",
+        None => String::from("http://127.0.0.1:8008/batches")
```

Comment:
Add a constant for this.  Something like:
const STRING: &'static DEFAULT_URL = "http://127.0.0.1:8008";
This will make it easy to change, for example, for use with Docker containers.